config-sources/zookeeper/pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.smallrye</groupId>
+    <artifactId>zookeeper-config-source</artifactId>
```

Comment:
I'm also wondering if we need an intermediate pom in /config-sources that changes the groupId to `io.smallrye.config-sources` and then this artifact inherits that groupId and has an artifactId of `zookeeper`?